examples/fungible/src/contract.rs

```diff
@@ -54,16 +53,16 @@ impl Contract for FungibleToken {
 
     async fn execute_operation(
         &mut self,
-        context: &OperationContext,
         operation: Self::Operation,
+        mut runtime: OperationRuntime,
```

Comment:
Can we keep the runtime as second argument?

This `mut` may raise some eyebrows. Not sure what would be the most convenient for users between `runtime: OperationRuntime`, `runtime: &OperationRuntime` and `runtime: &mut OperationRuntime`

Reply:
Done.